pkg/common/vclib/utils.go

```diff
@@ -206,5 +206,16 @@ func VerifyVolumePathsForVMDevices(vmDevices object.VirtualDeviceList, volPaths
 			}
 		}
 	}
+}
 
+func ExistsInList(needle string, haystack []string, caseSensitive bool) bool {
+	for _, straw := range haystack {
+		if caseSensitive && straw == needle {
+			return true
+		} else if strings.ToLower(straw) == strings.ToLower(needle) {
```

Comment:
`strings.EqualFold(straw, needle)`

Reply:
See an [explanation](https://github.com/akutz/sortfold) as to why `EqualFold` should almost always be used for case-insensitive string comparisons

Reply:
Will fix